profile/README.md

```diff
@@ -2,14 +2,14 @@
 
 ![An illustration showing a variety of differently themed Octocats. Monuments from different cities are indicated in the background like the Space Needle, Berlin Fernsehturm and Transamerica Pyramid.](https://user-images.githubusercontent.com/3369400/133268513-5bfe2f93-4402-42c9-a403-81c9e86934b6.jpeg)
 
-Yes, we are building GitHub on GitHub. In fact, we’ve been doing this since **October 19th, 2007**. That's when we made our first commit. Since then we pushed **over 2.5 million commits**, opened **over 1 million issues**, submitted roughly **650k pull requests** across **4357 repositories** from over **50 countries** 🤯. But that's just us. We are proud  to be part of the work of millions of developers, companies and robots across the solar system 🪐. Yes, [Robots](https://github.com/readme/featured/nasa-ingenuity-helicopter)!
+Yes, we are building GitHub on GitHub. In fact, we’ve been doing this since **October 19th, 2007**. That's when we made our first commit. Since then we pushed **over 6.3 million commits**, opened **over 3.3 million issues**, submitted roughly **3 million pull requests** across **9535 repositories** from over **67 countries** 🤯. But that's just us. We are proud to be part of the work of millions of developers, companies and robots across the solar system 🪐. Yes, [Robots](https://github.com/readme/featured/nasa-ingenuity-helicopter)!
 
 ### 🍿 An interconnected community
 
 The open source community is the 💗 heart of GitHub and fundamental to how we build software today. See for yourself:
 
-- [GitHub Sponsors](https://github.com/sponsors) helped support more than **5k** individuals and projects around the world 🌍
-- Open source projects on GitHub received a stunning **218 million** contributions 🚀 in the last year alone
+- [GitHub Sponsors](https://github.com/sponsors) helped support more than **17k** individuals and projects around the world 🌍
```

Comment:
We don't share the number of individuals and projects anymore. Instead, we can say that $40M+ has been invested in projects spanning 103+ regions.